content/applications/services/helpdesk/overview/sla.rst

```diff
@@ -178,7 +185,7 @@ corner above the chart.
 
 .. image:: sla/sla-report-cohort.png
    :align: center
-   :alt: View of the SLA status analysis report in cohort view
+   :alt: View of the SLA status analysis report in cohort view.
```

Comment:
For line 183, I'd add SLA before the 'Status Analysis' since that's the full name.